### ISSUE TITLE
Can we not add multiple word in label with space?

### ISSUE THREAD

**Issue by DSGourav** [Nukleon84/pyflowsheet#2]

**Comment by LarsH** [comment]
Yeah, it seems to be an issue with the internal SVG validator. However, I managed to bypass the issue by using nonbreaking spaces, Unicode character 0x00a0, like this:

```
pump3 = Pump("Pump\u00a03", "Pump 3", position = (600,90))
```
![image](https://github.com/Nukleon84/pyflowsheet/assets/1294050/c94bf0a4-292c-47be-b8bb-6f5df3554dea)
